application/models/UserGroup.php

```diff
@@ -590,7 +590,8 @@ public function sendUserEmails($ugid, $subject, $body, $copy)
             if ($emaiLResult['success']) {
                 $msgToUser .= gT('Sending successful') . "<br>";
             } else {
-                $msgToUser .= gT('Error: ') . $emaiLResult['msg'] . "<br>";
+                throw new Exception("Failed to send mail");
+                // $msgToUser .= gT('Error: ') . $emaiLResult['msg'] . "<br>";
```

Comment:
No reason to leave it in the code, @jack-han-115 . :) If we need to find it, it's in the commit history.